File: CodeTest/Java/BaekJoon/Class2_11_2869.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;


public class Class2_11_2869 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");

        int up = Integer.parseInt(st.nextToken());
        int down = Integer.parseInt(st.nextToken());
        int high = Integer.parseInt(st.nextToken());

        int day = (int)(high-up)/(up-down);
        int sign = (high-up)%(up-down);

        if(sign == 0) {
            System.out.println(day+1);
        } else {
            System.out.println(day+2);
        }
        
        /*
        * Scanner 사용시에 입출력 소요 시간이 너무 많음
        Scanner sc = new Scanner(System.in);

        int up = sc.nextInt();
        int down = sc.nextInt();
        int high = sc.nextInt();
        int day = 0;
        int sign = 0;
                
        day = (int)(high-up)/(up-down);
        sign = (high-up)%(up-down);

        if(sign == 0) {
            System.out.println(day+1);
        } else if(sign != 0) {
            System.out.println(day+2);
        }
        */

    }
}

File: CodeTest/Python/BaekJoon/Class2_11_2869.py
import sys
import datetime # 코드 소요 시간 측정하기 위함



# 코드 시작시 시간 측정
start_time = datetime.datetime.now()

up, down, high = map(int, sys.stdin.readline().split())

# high-up -> 며칠간의 반복 후 어떠한 값이 되었고, up을 한번 진행하면 high를 0보다 작거나 같은 수를 만드는 경우를 구한다
# up-down 이 high-up보다 크거나 같아지는 경우
# (up-down)*i >= high-up
# i >= (high-up) / (up-down)
day = (high-up)//(up-down)
sign = (high-up)%(up-down)  # up을 했을때, 딱 0이 나오면 1회 늘고, 딱 맞지 않으면 2회 추가

if sign == 0: print(day+1)
elif sign != 0: print(day+2)


# 코드 종료 후 시간 측정
end_time = datetime.datetime.now()
# seconds 사용해서 초를 구할수 있지만, 정수형 리턴으로 자세한 시간을 구하려면 아래 방식 사용
print((end_time - start_time).microseconds/1000/1000,'초')

"""
# 극단적인 경우 실행 시간이 너무 오래걸림
# 빅오를 더 낮춰야함
day = 0

while high >= 0:
    day += 1

    high -= up

    if high<=0:
        break
    else:
        high += down
print(day)
"""